docs/deploy-cluster.md

```diff
@@ -0,0 +1,150 @@
+---
+
+This tutorial provides an introduction to deploy `Nebula` cluster.
+
+---
+
+#### Download and install package
+
+First at all, you can download rpm or deb from [Here]().
+
+For `CentOS`:
+
+```
+rpm -ivh nebula-{VERSION}-0.el7.centos.x86_64.rpm
+```
+
+For `Ubuntu`:
+
+```
+dpkg -i nebula-{VERSION}-0.el7.centos.x86_64.deb
+```
+
+By default, the config files are under `/usr/local/etc`, you should modify the `meta_server_addrs` to set the Meta Server's address. 
+
+In order to enable multi copy Meta service, you should set the meta addresses split by comma into `meta_server_addrs`.
+
+Using `data_path` to set `Meta` and `Storage`'s underlying storage directory.
+
+***
+
+#### StartUp Nebula Cluster
+
+Currently, we have support `scripts/services.sh` to management the nebula cluster.
+
+You can `start`, `stop` and `restart` the cluster using this script.
+
+It's looks like the following command:
+
+```
+scripts/services.sh <start|stop|restart|status|kill>
+```
+
+The metas, storages and graphs contain the host of them.
+
+***
+
+#### Config reference
+
+**Meta Service** support the following config properties.
+
+Property Name               | Default Value            | Description
+--------------------------- | ------------------------ | -----------
+`port`                      | 45500                    | Meta daemon listening port.
+`reuse_port`                | true                     | Whether to turn on the SO_REUSEPORT option.
+`data_path`                 | ""                       | Root data path.
+`peers`                     | ""                       | It is a list of IPs split by comma, the ips number equals replica number. If empty, it means replica is 1.
+`local_ip`                  | ""                       | Local ip speicified for NetworkUtils::getLocalIP.
+`num_io_threads`            | 16                       | Number of IO threads.
+`meta_http_thread_num`      | 3                        | Number of meta daemon's http thread.
+`num_worker_threads`        | 32                       | Number of workers.
+`part_man_type`             | memory                   | memory, meta.
+`pid_file`                  | "pids/nebula-metad.pid"  | File to hold the process id.
+`daemonize`                 | true                     | Whether run as a daemon process.
+`cluster_id`                | 0                        | A unique id for each cluster.
+`putTryNum`                 | 10                       | Try num of store clusterId to kvstore.
```

Comment:
This description is somehow confusing...

Reply:
I lost attention on it.

`Number of attempts to generate cluster ID` would be better ?

Reply:
Yes, much more understandable 👍